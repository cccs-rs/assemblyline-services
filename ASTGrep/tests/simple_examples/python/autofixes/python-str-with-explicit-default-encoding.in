str(x, "someencoding")
str("aaaa", "utf8")
str(x, "UtF-8")
