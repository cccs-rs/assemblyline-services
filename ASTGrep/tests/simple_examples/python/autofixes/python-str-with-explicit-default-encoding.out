str(x, "someencoding")
"aaaa"
str(x)
